server_addon/houdini/server/settings/publish.py

```diff
@@ -20,6 +20,39 @@ class CollectChunkSizeModel(BaseSettingsModel):
         title="Frames Per Task")
 
 
+def product_types_enum():
+    return [
+        {"value": "camera", "label": "Camera (Abc)"},
+        {"value": "pointcache", "label": "PointCache (Abc)/PointCache (Bgeo)"},
+        {"value": "review", "staticMesh": "Static Mesh (FBX)"},
+        {"value": "usd", "label": "USD (experimental)"},
+        {"value": "vdbcache", "label": "VDB Cache"},
+        {"value": "imagesequence", "label": "Composite (Image Sequence)"},
+        {"value": "review", "review": "Review"},
+        {"value": "ass", "label": "Arnold ASS"},
+        # {"value": "arnold_rop", "label": "Arnold ROP"},
+        {"value": "mantraifd", "label": "Mantra IFD"},
+        # {"value": "mantra_rop", "label": "Mantra ROP"},
+        {"value": "redshiftproxy", "label": "Redshift Proxy"},
+        # {"value": "redshift_rop", "label": "Redshift ROP"},
+        # {"value": "karma_rop", "label": "Karma ROP"},
+        # {"value": "vray_rop", "label": "VRay ROP"},
```

Comment:
Are some here still commented on purpsose? If so, should we add a comment as to why?

Reply:
It was for testing purposes.